### DISCUSSION TITLE
Typo in UPDATE documentation?

### DISCUSSION THREAD

**Discussion by jenstroeger** [sqlalchemy/sqlalchemy#10883] [Usage Questions] [answered]
In the second example of the [The update() SQL Expression Construct](https://docs.sqlalchemy.org/en/20/tutorial/data_update.html#the-update-sql-expression-construct) docs, the Python code is

https://github.com/sqlalchemy/sqlalchemy/blob/2c8a05244a8f7919c3954e0b6e1efaa996792772/doc/build/tutorial/data_update.rst?plain=1#L74-L76

but I think should be `user_table.name`.

In my case, I get a
```
AttributeError: type object 'SomeTable' has no attribute 'c'
```
with the example code, but it works after removing the `.c`.

**Comment by CaselIT** [comment, marked as answer]
Hi,

The documentation seems correct, since it's using a table, defined a few pages before, here
https://github.com/sqlalchemy/sqlalchemy/blob/2c8a05244a8f7919c3954e0b6e1efaa996792772/doc/build/tutorial/metadata.rst?plain=1#L80-L87

You seem to be using an ORM Model, in that case since it's a class, you should use name directly, without the `.c`.
